### PR TITLE
Improved performance

### PR DESCRIPTION
**Pull request by xbolva00**

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -107,7 +107,7 @@ def load_rooms(rooms, argv):
 
     if rooms.fl:
         rooms.fr = rooms.fl * 100
-        rooms.to = (rooms.fr + 100) - 1
+        rooms.to = rooms.fr + 22 # 22 rooms on the floor
```
Comment (tammar96) [inline main.py]: on PPV are 41 rooms

**Comment by tammar96** [comment]
Squash commits into one

File: finder.py [hunk 2]
```diff
@@ -28,7 +28,12 @@ def get(block, room):
 
     block = block.lower()
 
-    r = requests.get('http://kn.vutbr.cz/search/index.html?str='+str(block)+'-'+str(room))
+    room = str(room)
+    block = str(block)
+    if block.startswith('a') or block.startswith('d'):
```
Comment (tammar96) [inline finder.py]: purkyne?

Comment (xbolva00) [reply finder.py]: manesky :D

Comment (tammar96) [reply finder.py]: or block.startswith('b')

**Pull request merged**